docs/man_pages/publishing/publish-add.md

```diff
@@ -0,0 +1,18 @@
+publish add
+==========
+
+Usage | Syntax
+------|-------
+General | `$ appbuilder publish add [<Name>] [<PublishUrl>]`
```

Comment:
is the Name mandatory parameter? What happens in case you do not pass it? This should be documented.